pkg/gql/query/transactions.go

```diff
@@ -80,6 +81,7 @@ func newQueryTx(tx core.ContractCall, blockHash []byte, timestamp int64, blockhe
 	qd.GasPrice = decoded.Fee.GasPrice
 
 	qd.GasSpent = tx.GasSpent()
+	qd.Raw = tx.StandardTx().Data
```

Comment:
Could you update the gql README with an example usage of the new field?

Reply:
Added to the example with the full supported fields list